doc/source/users_guide/using-clm-tools/using-ctsm-pylib.rst

```diff
@@ -0,0 +1,14 @@
+
+.. include:: ../substitutions.rst
+.. _using-ctsm-pylib:
+
+Installing the CTSM Python environment
```

Comment:
- [ ] Make sure to link to this page in the `subset_data` docs (coming in on #3002).